src/ui/Views/Course/Variants.cshtml

```diff
@@ -25,35 +25,55 @@
           <h3 class="course-parts__title">Information from UCAS</h3>
 
           <dl class="govuk-list govuk-list--description">
-            @if (!string.IsNullOrEmpty(@Model.Course.Name)) {
-              <dt>Name:</dt>
-              <dd>@Model.Course.Name</dd>
+            @if (Model.Course.Status.Equals("Running", StringComparison.InvariantCultureIgnoreCase))
+            {
+              if (!string.IsNullOrEmpty(@Model.Course.Name))
+              {
+                <dt>Name:</dt>
+                <dd>@Model.Course.Name</dd>
+              }
+            }
+            @*else
+            {
+              @Html.Partial("~/Views/Shared/_Alerts.cshtml")
```

Comment:
Is this meant to be commented out?

Reply:
I can't see what this block is doing. Trying to find a "running" course to check

Reply:
Removed this block